client/src/components/RelatedItems/YourOutfit.jsx

```diff
@@ -1,18 +1,49 @@
-import React from 'react';
+import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import AddOutfit from './AddOutfit.jsx';
 import ProductCard from './ProductCard.jsx';
 import './relatedItemsStyle.css';
 
 function YourOutfit({outfitProductsInfo}) {
+  const [currentCardIndex, setCurrentCardIndex] = useState(0);
+  const carrouseLength = outfitProductsInfo.length;
+
+  let previousButtonOnClick = () => {
+    if (currentCardIndex > 0) {
+      setCurrentCardIndex(currentCardIndex - 1);
+    }
+  };
+
+  let nextButtonOnClick = () => {
+    if (currentCardIndex < (carrouseLength - 3)) {
+      setCurrentCardIndex(currentCardIndex + 1);
+    }
+  };
+
   return (
     <div>
       <p>YOUR OUTFIT</p>
-      <div className="main-gallery js-flickity" data-flickity='{"pageDots": false, "cellAlign": "left"}'>
-        <AddOutfit />
-        {outfitProductsInfo.map((outfitProductInfo, index) => {
-          return (<ProductCard key={index} outfitProductInfo={outfitProductInfo} />);
-        })}
+      <div className="carousel">
+        <div className="carousel-wrapper">
+          {currentCardIndex > 0 &&
+            <button onClick={previousButtonOnClick} className="previous-button">
+              &lt;
+            </button>
+          }
+          <div className="carousel-inner">
+            <div className="carousel-card" style={{ transform: `translateX(-${currentCardIndex * (100 / 3.81)}%)` }}>
```

Comment:
This is interesting. You're positioning the div based on the index --- very cool